src/deployments.md

```diff
@@ -53,13 +56,14 @@ You will need:
 ![Palisade JVM Deployment](../img/JVM-Deployment.png)
```

Comment:
Images need updating to include the data service interactions, since it is only currently showing the registration process and people will think that is all Palisade does and not realise that we also filter the data as well

Reply:
added